scalameta/contrib/src/main/scala/scala/meta/contrib/implicits/CommentExtensions.scala

```diff
@@ -0,0 +1,43 @@
+package scala.meta.contrib.implicits
+
+import scala.meta._
+import scala.meta.contrib.{DocToken, ScaladocParser}
+import scala.meta.tokens.Token.Comment
+
+trait CommentExtensions {
+
+  private[this] val scaladocSymbols = Seq('/', '*', ' ')
+
+  implicit class XtensionCommentOps(c: Comment) {
+
+    private[this] val rawSyntax: String = c.show[Syntax].trim
+
+    @inline
+    def isScaladoc: Boolean = {
```

Comment:
I recommend we keep the implementations outside the implicit class as static methods on a `CommentOps` object, just like the TreeExtensions.